pids_test.go

```diff
@@ -0,0 +1,169 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package cgroups
+
+import (
+	"encoding/hex"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"testing"
+)
+
+func TestPids(t *testing.T) {
+	mock, err := newMock()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer mock.delete()
+	pids := NewPids(mock.root)
+	if pids == nil {
+		t.Error("pids is nil")
+		return
+	}
+	resources := specs.LinuxResources{}
+	resources.Pids = &specs.LinuxPids{}
+	resources.Pids.Limit = int64(10)
+	err = pids.Create("test", &resources)
+	if err != nil {
+		t.Error(err)
```

Comment:
You can use `t.Fatal` here so you don't have to have a `return` in every block

Reply:
updated